cypress/integration/e2e/machine_spec.js

```diff
@@ -42,7 +42,8 @@ describe("State Machine Spec", () => {
             cy.wait(2000);
             cy.get(".component-drawer-svg-container[draggable='true']").first().click().drag("#cy-canvas-container", {force: true})
             // Perform delete action
-            cy.get("#component-delete").click();
+            cy.get("#component-delete",{timeout: 3000}).click();
```

Comment:
@harkiratsm Time is one of the solution. But we are already doing this many times, which is causing us to increase our build minutes. We have limited build minutes, for now, I guess. // @Abhishek-kumar09 Your views.

Reply:
Are you sure @Chadha93 . Because {timeout} doesn't wait but stops once the item is visible, by the way the default timer is 4s and we are using 3 seconds here. It won't add any benefit here I think.

Reply:
```suggestion
            cy.get("#component-delete").click();
```